### DISCUSSION TITLE
Consul Config with Helidon

### DISCUSSION THREAD

**Discussion by PabloCaiza** [helidon-io/helidon#4193] [Q&A] [answered]
Hello, I have a question. Can I integrate consul or spring cloud config to configure my Helidon application?

**Comment by arjav-desai** [comment, marked as answer]
Apologies for delay in response. We have implemented etcd as config source, see https://github.com/helidon-io/helidon/tree/helidon-3.x/config and provide SPI to write your own https://helidon.io/docs/latest/#/mp/config/advanced-configuration#_creating_custom_config_sources
